backend/api/EventHandlers/InspectionFindingEventHandler.cs

```diff
@@ -23,12 +29,150 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
                     logger.LogInformation("Found {count} inspection findings in the last {interval}.", inspectionFindings.Count, _timeSpan);
 
+                    if (inspectionFindings.Count > 0)
+                    {
+                        var findingsList = await GenerateFindingsList(inspectionFindings);
+
+                        string messageString = GenerateReportFromFindingsReportsList(findingsList);
+
+                        string adaptiveCardJson = GenerateAdaptiveCard(messageString);
+
+                        string url = GetWebhookURL("TeamsInspectionFindingsWebhook");
+
+                        var client = new HttpClient();
+                        client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+                        var content = new StringContent(adaptiveCardJson, Encoding.UTF8, "application/json");
+                        var response = await client.PostAsync(url, content, stoppingToken);
+                        //if (response.StatusCode == StatusCodes.Status201Created) ;
```

Comment:
It is probably good to include a check of the response here. If there is an error you can log it